pdf_craft/sequence/content.py

```
@@ -0,0 +1,38 @@
+
+def expand_text_in_content(
+        content: list[str | BlockMember | HTMLTag[BlockMember]],
+        expand: Callable[[str], Iterable[str | BlockMember]],
+) -> None:
+    for sub_content in _search_content(content):
+        i: int = 0
+        while i < len(sub_content):
+            part = sub_content[i]
+            if isinstance(part, str):
+                del sub_content[i]
+                for part in expand(part):
```

Ruff (0.14.8)

Loop control variable `part` overrides iterable it iterates

(B020)